test/integ_tests/test_cost_tracking.py

```diff
@@ -0,0 +1,97 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+from datetime import timedelta
+
+from braket.annealing import Problem, ProblemType
+from braket.aws import AwsDevice
+from braket.circuits import Circuit
+from braket.tracking import Tracker
+from braket.tracking.tracker import MIN_SIMULATOR_DURATION
+
+
```

Comment:
Could we add a test to assert that we can fetch the pricing values for all devices ? (fetch devices using AwsDevice.get_devices(), call search_prices for each price type applicable per device type and assert)